examples/compose/local/setup.sh

```diff
@@ -110,7 +111,7 @@ else
   password
 fi
 
-sudo docker-compose up -d
+sudo docker-compose up -d || sudo docker compose up -d
```

Comment:
Instead of doing this, factorize it so that this is tested and the command is known (rather than using `||`).

For example:

```bash
COMPOSE=""

if [[ $(docker compose version) ]]; then
  COMPOSE="docker compose"
elif [[ -x "$(command -v docker-compose)"]]; then
  COMPOSE="docker-compose"
else
  echo "You must install Docker Compose on your machine";
  exit 1
fi
```

```bash
sudo ${COMPOSE} up -d
```